src/SwedbankPay.Episerver.Checkout/SwedbankPayCheckoutService.cs

```diff
@@ -53,7 +54,7 @@ public class SwedbankPayCheckoutService : ISwedbankPayCheckoutService
         public virtual PaymentOrder CreateOrUpdatePaymentOrder(IOrderGroup orderGroup, string description,
             string consumerProfileRef = null)
         {
-            var orderId = orderGroup.Properties[Constants.SwedbankPayCheckoutOrderIdCartField]?.ToString();
+            var orderId = orderGroup.Properties[Constants.SwedbankPayOrderIdField]?.ToString();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'orderId'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-episerver-checkout/pullRequest?prid=5328029)